contributing.md

```diff
@@ -0,0 +1,163 @@
+# Contributing to pylabrad
+
+## Process
+
+The following instructions are for pylabrad contributors.
+Other users working in their own fork can will use whatever procedure they like for managing contributions.
+Once those contributions are ready to be included in the main pylabrad repository, they should file a pull request and allow the pylabrad contributors to carry out the rest of the steps.
+
```

Comment:
It would be good to have a high-level comment here that the development strategy is to keep a linear history on master, which is why we do merging the way we do, rather than using github's merge tool, for example. That may not mean anything to some people on a first read, but it's a good intro for folks who are more familiar with git or reading through this later.